src/main/java/school/lemon/changerequest/java/introduction/pr1/HammingDistance.java

```diff
@@ -2,6 +2,21 @@
 
 public class HammingDistance {
     public static int compute(String a, String a1) {
-        return -1;
-    }
-}
+        char[] aA = a.toCharArray();
+        char[] a1A1 = a1.toCharArray();
```

Comment:
There is no need for this 2 additional variables.
`String` has method `chatAt` that will given you character at some position.